<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VZMainWindow</class>
 <widget class="QMainWindow" name="VZMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>682</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1221</width>
    <height>662</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1221</width>
    <height>682</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>四川前途科技车牌识别</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblPicture">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>341</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPlatePic">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>280</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabResult">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>340</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>车牌</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>可信度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>颜色</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>宽高</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>方向</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>车牌文件</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>文件</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>通道</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>视频识别</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="lblVideo0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>300</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>902</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnFile">
       <property name="text">
        <string>识别所有文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSingleFile">
       <property name="text">
        <string>识别单一文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPreFile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>上一文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNextFile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>下一文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>清除所有</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAnalogCamera">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>模拟摄像机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAnalogCaptureFile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>模拟抓图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDigitalCamera">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>数字摄像机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnIPCCaptureFile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>数字抓图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnVideoFile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>视频文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStopVideoRecognize">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>停止视频识别</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblVideo1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>421</width>
      <height>300</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblVideo2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>421</width>
      <height>300</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblVideo3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>320</y>
      <width>421</width>
      <height>300</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>系统配置</string>
    </property>
    <addaction name="actParameter"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actParameter">
   <property name="text">
    <string>参数配置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
